cpp/dolfinx/fem/utils.h

```diff
@@ -418,6 +418,11 @@ Form<T, U> create_form_factory(
       ufcx_integral* integral
           = ufcx_form.form_integrals[integral_offsets[cell] + i];
       assert(integral);
+      const bool* _enabled_coefficients = integral->enabled_coefficients;
```

Comment:
Nothing wrong with this, but we should probably replace `bool` by `uint8_t` in UFCx.

Reply:
Could we do that in a separate PR, as this keeps the current interface (and could therefore in theory be backported).